src/control/security/grpc_certs.go

```diff
@@ -44,9 +44,16 @@ func GetServerTransportCredentials(cfg *TransportConfig) (credentials.TransportC
 	}
 
 	tlsConfig := tls.Config{
```

Comment:
An overall comment describing the rationale behind these configuration choices would be very helpful to make sure that future maintainers understand why the choices were made and help to inform future choices.